packages/iTwinUI-react/src/core/Toast/Toast.tsx

```diff
@@ -177,11 +177,11 @@ export const Toast = (props: ToastProps) => {
       in={visible}
       appear={true}
       unmountOnExit={true}
-      onEnter={(node) => {
+      onEnter={(node: HTMLElement) => {
```

Comment:
Why is this needed? Inline functions should infer types

Reply:
no idea, but I was getting errors and only on those 2 :D